machine_learning_hep/globalfitter.py

```diff
@@ -18,38 +18,8 @@
 
 import math
 from array import array
-from ROOT import TF1 # pylint: disable=import-error,no-name-in-module
-from ROOT import gStyle # pylint: disable=import-error,no-name-in-module
-from ROOT import kBlue, kGray # pylint: disable=import-error,no-name-in-module
-from ROOT import TCanvas, TPaveText, Double # pylint: disable=import-error,no-name-in-module
-from ROOT import TVirtualFitter # pylint: disable=import-error,no-name-in-module
-
-def gaus_func(sgnfunc):
-    if sgnfunc == "kGaus":
-        sigval = "[0]/(sqrt(2.*pi))/[2]*(exp(-(x-[1])*(x-[1])/2./[2]/[2]))"
-    return sigval
-
-def pol1_func(xval, par):
```

Comment:
The stripped versions of the fit functions (here used for "back_fit_new" vs. "back_refit") are not necessary.